Cargo.toml

```diff
@@ -19,7 +19,7 @@ appveyor = []
 travis = []
 
 [dependencies]
-libc = "0.2"
+libc = ">=0.2.34"
```

Comment:
You can actually remove this dependency completely. All usage of `libc` has now been moved to the sub-crates `pnet_sys` and `pnet_datalink`. Then also remember to remove the `extern crate libc;` in `src/lib.rs`.